subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/BasicGroupedTaskLoggingFunctionalSpec.groovy

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.logging
+
+import org.gradle.integtests.fixtures.AbstractConsoleFunctionalSpec
+
+class BasicGroupedTaskLoggingFunctionalSpec extends AbstractConsoleFunctionalSpec {
+
+    def "Parallel tasks logs are still grouped by task"() {
+        given:
+        settingsFile << "include '1', '2', '3'"
+
+        buildFile << """
+            subprojects {
+                task log { doFirst { logger.quiet "Output from " + project.name } }
+            }
+        """
+
+        when:
+        // run in quiet mode just to make it easier to see
+        executer.withArgument('--parallel')
```

Comment:
It doesn't matter.  And if I leave parallel off, it will run both ways on CI right?